webdav.go

```diff
@@ -34,6 +38,20 @@ func init() {
 	caddy.RegisterModule(WebDAV{})
 }
 
+// ETag calculation implementation from caddy/modules/caddyhttp/fileserver/staticfiles.go
```

Comment:
Might be good to explain why it's the same calculation in particular.

Maybe we should export it from the fileserver package?

Reply:
I've expanded the comments. Webdav is still an out of tree module, so I think there's no hurry to export the calculation function.